templates/comments/list.html

```diff
@@ -60,7 +60,7 @@ <h2><i style="padding-right: 0.3em" class="fa fa-comments"></i>{{ _('Comments')
                                     <a href="#comment-{{ node.id }}" title="{{ _('Link') }}" class="comment-link">
                                     <i class="fa fa-link fa-fw"></i>
                                 </a>
-                                    {% if logged_in %}
+                                    {% if logged_in and not comment_lock %}
```

Comment:
Perhaps the list should display a message to the effect of "Comments for this page are disabled." rather than just removing the comment list, so prevent users from thinking their absence is a bug.